cmd/cadvisor.go

```diff
@@ -140,9 +142,40 @@ func (ml *metricSetValue) Set(value string) error {
 	return nil
 }
 
+type DenyList map[string]struct{}
```

Comment:
As far as I understand this type is unused and can be removed.

Reply:
This type is used as follows:
![image](https://user-images.githubusercontent.com/27987483/108331147-7057b380-7209-11eb-825c-23d75953a578.png)
It is created for flag.var:
![image](https://user-images.githubusercontent.com/27987483/108331285-954c2680-7209-11eb-866d-e611e9a519fe.png)

Reply:
Can you point me at location in code where any function of this struct is used? As far as I understand you use it as plain map[string]struct{} so there is no point in declaring new type and its functions.

Reply:
Hi,iwankgb,I have made some new change.review it please.